tests/Doctrine/Tests/DBAL/Functional/Types/LanguageDoubleTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Functional\Types;
+
+use const LC_ALL;
+use function setlocale;
+
+class LanguageDoubleTest extends DoubleTest
+{
+    protected function setUp() : void
+    {
+        setlocale(LC_ALL, 'de_DE.UTF-8', 'de_DE', 'de', 'ge');
```

Comment:
What if neither of the locales is available at the runtime? Should there be any teardown logic in the test?

Reply:
@morozov 

I can make a tear down.

For travis, every run has the de_DE language installed, but I suppose I could try to add detection logic here if it works.